composer-dependency-analyser.php

```diff
@@ -0,0 +1,15 @@
+<?php declare(strict_types = 1);
+
+use ShipMonk\ComposerDependencyAnalyser\Config\Configuration;
+use ShipMonk\ComposerDependencyAnalyser\Config\ErrorType;
+
+$config = new Configuration();
+
+return $config
+    ->addPathToScan(__DIR__ . '/build/config', false)
+    ->ignoreErrorsOnPackage('phpunit/phpunit', [ErrorType::DEV_DEPENDENCY_IN_PROD]) // prepared test tooling
+    ->ignoreErrorsOnPaths([
+        __DIR__ . '/stubs',
+        __DIR__ . '/tests',
+        __DIR__ . '/rules-tests',
+    ], [ErrorType::UNKNOWN_CLASS]);
```

Comment:
Here is original report of those. I assume you want to ignore unknown symbols in tests.

![image](https://github.com/rectorphp/rector-src/assets/1993453/29e4630a-90d6-417c-90f3-019e741a74ab)